--- extended_bin	2019-03-22 23:51:12.105140739 +0300
+++ extended_bin2	2019-06-14 23:05:29.344069349 +0300
@@ -160,7 +160,7 @@
 relx_rem_sh() {
     # Generate a unique id used to allow multiple remsh to the same node
     # transparently
-    id="remsh$(relx_gen_id)-${NAME}"
+    id="remoteshell-${NAME}"
 
     # Get the node's ticktime so that we use the same thing.
     TICKTIME="$(relx_nodetool rpcterms net_kernel get_net_ticktime)"
